### ISSUE TITLE
Search results do not include @context

### ISSUE THREAD

**Issue by siuc-nate** [CredentialEngine/CredentialRegistry#217]
It would be handy if the search results (the `__payload` data, specifically) had the `@context` injected into it, just as we do with the same data when retrieved via its `/resources/` URI.  This would ensure that the consumer of the data has the correct context for each result.

**Comment by siuc-nate** [comment]
None jump to my mind, but I'm not familiar with the internals of the relevant code. I would assume that it probably makes sense to do the injection at index time rather than on-demand.

**Comment by siuc-nate** [comment]
It makes sense, but it sounds a bit more complex than adding the `@context` to the existing `__payload` data - and you are correct that it is probably too close to the event to make such a substantial change.

**Comment by rsaksida** [comment]
From a backend programming perspective, it is definitely more complex, but wouldn't really be a lot of work (unless I run into something unexpected). From a client side perspective I think it'd simplify things. It'd definitely require more time, though.

I think we have time to implement `@context` inside `__payload` before the event, but it's important to resolve the hardware limitations we have. Right now it takes a few days to reindex the envelopes as a graph.

**Comment by cwd-mparsons** [comment]
@rsaksida I see that the context is still not part of the search results. Is this solution still held up by the performance issues?

**Comment by rsaksida** [comment]
Hmm, I don't remember my thought process at the time - I think it was to preserve the original document inside `__payload` - but I did implement this. However, it's slightly different from what's specified in the issue. The `@context` property is available in the graph node, just not inside `__payload`. You can request that property along with `__payload`.

For example:

```groovy
g.V()
  .has('__graph_id', 'ce-f5680245-45af-4619-b35b-3c68a57407eb')
  .valueMap('__payload', '@context')
```

Will return both the `__payload` and the `@context` for that resource ctid.

Can you work with this?

**Comment by siuc-nate** [comment]
I will investigate injecting this into the results as part of the search API.

**Comment by siuc-nate** [comment]
This issue is no longer relevant. Closing.
